libraries/EEPROM/src/utility/stm32_eeprom.c

```diff
@@ -141,12 +142,34 @@ void eeprom_write_byte(uint32_t pos, uint8_t value)
   }
 #else
   eeprom_buffered_write_byte(pos, value);
-  eeprom_buffer_flush();
+  if(do_auto_flush)
+    eeprom_buffer_flush();
 #endif /* _EEPROM_BASE */
 }
 
 #if !defined(DATA_EEPROM_BASE)
 
+/**
+  * @brief  This function disables the auto flush in eeprom_write_byte - user will need to call eeprom_buffer_flush() explicitly
+  *         It affects write operations through eeprom_write_byte() explicit call, EEPtr, EERef and EEPROMClass classes
+  * @param  none
+  * @retval none
+  */
+void disable_auto_flush(void)
+{
+  do_auto_flush = false;
+}
+
+/**
+  * @brief  This function enables the auto flush in eeprom_write_byte - it is enabled by default
+  * @param  none
+  * @retval none
+  */
+void enable_auto_flush(void)
+{
+  do_auto_flush = true;
+}
+
```

Comment:
Those functions are not defined in .h.
And can be grouped:
```
void eeprom_auto_flush(bool enable)
{
  do_auto_flush = enable;
}
```

Reply:
I did a test build, it built just fine for me. But I agree it's better to group them into one function.